src/backend/templates/rooms.hbs

```diff
@@ -57,7 +57,7 @@
         {{#timeList}}
           <span class="date-pill"><a href="./rooms.html#{{slug}}"> {{date}} </a></span>
         {{/timeList}}
-        <span style = "display:inline-block; padding-top:6px;"> 
+        <span class="show-button-filter" > 
```

Comment:
@aayusharora sorry about the redundant line , I may have pasted it by mistake . I will be more careful from now on.

Thank you! :)